calcom/cal.com
# fix: Duplicate calendar events

## What does this PR do?



This PR aims to fix duplicate calendar events being created for team members and dynamic event participants. A full explanation of what happened can be found here: https://threads.com/34535947634?s=2qEeCrUqS5rpJPa2fo7kjU

Fixes # (issue)



## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change





## How should this be tested?



- Create a team of users
- Connect different GCals to those users
- **IMPORTANT** those user's emails should be the emails being used to connect the calendar
- Now create one more user that is connected to Outlook
- Book a collective event type with all three users
    - There should be only one event created in GCal and one event in Outlook
- Repeat this again for round robin and a dynamic event with those three users

## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
